### DISCUSSION TITLE
Super-deep AGS

### DISCUSSION THREAD

**Discussion by damogles** [NatLabRockies/GEOPHIRES-X#283] [General]
Really appreciate the effort and the quality of this open-source simulation! Thank you!

I am simulating a closed-loop super-deep geothermal system (~10km depth with multiple laterals). I am wondering how the model behaves in those extreme conditions? My main concern is the dependability of the output value. 
- how is supercritical condition treated?
- are there limits to extrapolation beyond the research-validated models which are in ~5km depth?
- how does maximum temperature influence the model? why is it not a function of depth, gradient, and ambient temperature?

I am curious about the exchange. And again, thank you!

**Comment by softwareengineerprogrammer** [comment]
Issues created during our discussion today, for posterity:
1. https://github.com/NREL/GEOPHIRES-X/issues/278?title=%22Total+Nonvertical+Length%22+is+actually+per+section
2. https://github.com/NREL/GEOPHIRES-X/issues/279?title=Total+length+not+reflected+in+AGS+input
3. https://github.com/NREL/GEOPHIRES-X/issues/280?title=AGS+does+not+enforce+Maximum+Temperature
4. https://github.com/NREL/GEOPHIRES-X/issues/281?title=AGS:+Math+domain+error+for+supercritical+ORC+plant
5. https://github.com/NREL/GEOPHIRES-X/issues/282?title=Negative+pumping+depth+warning?

---

> how is supercritical condition treated?

> are there limits to extrapolation beyond the research-validated models which are in ~5km depth?

High-level answer: these handled by Wanju extrapolations (currently) and/or SBT (near-future); @malcolm-dsider can possibly provide additional context.
